### PR TITLE
Feature/user experience bugs

### PR DESCRIPTION
**Pull request by collinkallery**
### Type of change made:

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling (Styling to new or existing files)

### Detailed Description
- Fixed labels on CreateEvent, put in cost input, remove new address inputs, refactor location methods, add ability to share event. 

### Why is this change required? What problem does it solve?
- This allows a user to share an event and choose labels when they are creating an event. We also removed the ability to create a new location. 
### Was any previously written code changed?
- Yes, we removed the location adding feature because we don't have time before submitting the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "react": "^16.13.1",
+    "react-copy-to-clipboard": "^5.0.2",
```
Comment (collinkallery) [inline package.json:10]: Will need to remove this unused dependency.

File: src/Containers/CreateEvent/CreateEvent.js [hunk 2]
```diff
@@ -255,6 +268,7 @@ const CreateEvent = (props) => {
   }
 
   const updateFormState = (e) => {
+    let label = ''
```
Comment (collinkallery) [inline src/Containers/CreateEvent/CreateEvent.js:271]: Will need to remove this as well

File: src/Containers/CreateEvent/CreateEvent.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import Header from '../../Components/Header/Header.js';
+import { NavLink } from 'react-router-dom';
```
Comment (TravisBorgsmiller) [inline src/Containers/CreateEvent/CreateEvent.js:4]: Does this add special styling specifically when we create an event?

Comment (KCWill) [reply src/Containers/CreateEvent/CreateEvent.js:4]: Yeah, the style-components is a package to let you modify the CSS in the component file. Collie showed me how to do it!

**Pull request merged**